__tests__/Unit/Components/Tasks/TaskDetails.test.tsx

```diff
@@ -659,3 +643,78 @@ describe('TaskDependency', () => {
         });
     });
 });
+
+jest.mock('next/router', () => ({
+    useRouter() {
+        return {
+            query: {
+                dev: 'true',
+            },
+            push: mockNavigateToUpdateProgressPage,
+        };
+    },
+}));
+
+describe('Details component', () => {
+    it('Renders the detail type and value correctly', () => {
+        renderWithRouter(<Details detailType="Assignee" value="John Doe" />);
+        const detailType = screen.getByText('Assignee:');
+        const detailValue = screen.getByText('John Doe');
+        expect(detailType).toBeInTheDocument();
+        expect(detailValue).toBeInTheDocument();
+    });
+
+    it('Renders a GitHub link correctly', () => {
+        const githubIssueUrl = 'https://github.com/example/repo/issues/123';
+        renderWithRouter(<Details detailType="Link" value={githubIssueUrl} />);
+        const link = screen.getByRole('link', { name: /Open GitHub Issue/i });
+        expect(link).toBeInTheDocument();
+        expect(link).toHaveAttribute('href', githubIssueUrl);
+    });
+
+    it('Does not display tooltip on hover when timestamp is not provided', async () => {
+        render(<Details detailType="Started On" value="N/A" />);
+        const detailValue = screen.getByText('Started:');
+        fireEvent.mouseOver(detailValue);
+        const tooltip = screen.queryByText('N/A', { selector: '.tooltip' });
+        expect(tooltip).not.toBeInTheDocument();
+    });
+
+    it('Renders an extension request icon for Ends On with URL', () => {
+        const url = 'https://example.com';
```

Comment:
can we have this fixtures in fixtures file?

Reply:
Didn't fully understand what are you trying to say?
Also, I didn't find any folder named fixtures

Reply:
It's present id this folder
website-status\__mocks__\db